understanding/21/target-size-enhanced.html

```diff
@@ -17,19 +17,19 @@ <h2>In brief</h2>
   	</section>
   	<section id="intent">
   		<h2>Intent</h2>
-  		<p>The intent of this success criterion is to help users who may have trouble activating a small target because of hand tremors, limited dexterity or other reasons. If the target is too small, it may be difficult to aim at the target. Mice and similar pointing devices can be hard to use for these users, and a larger target will help them greatly in having positive outcomes on the web page.</p>
+  		<p>The intent of this success criterion is to help users who may have trouble activating a small target because of hand tremors, limited dexterity, or other reasons. If the target is too small, it may be difficult to aim at the target. Mice and similar pointing devices can be hard to use for these users, and a larger target will help them greatly in having positive outcomes on the web page.</p>
       <p>Touch is particularly problematic as it is an input mechanism with coarse precision. Users lack the same level of fine control as on inputs such as a mouse or stylus. A finger is larger than a mouse pointer, and generally obstructs the user's view of the precise location on the screen that is being touched/activated.</p>
-      <p>The issue can be further complicated for responsive/mobile sites which need to accommodate different types of fine and coarse inputs (e.g. a site that can be accessed both on a traditional desktop/laptop with a mouse, as well as on a tablet or mobile phone with a touch screen).</p>
+      <p>The issue can be further complicated for responsive and mobile sites that need to accommodate different types of fine and coarse inputs (for example, a site that can be accessed on a traditional desktop or laptop with a mouse, as well as on a tablet or mobile phone with a touch screen).</p>
       <p>While this criterion defines a minimum target size, it is recommended that larger sizes are used to reduce the possibility of unintentional actions. This is particularly relevant if any of the following are true:</p>
   	  <ul>
         <li>the control is used frequently;</li>
         <li>the result of the interaction cannot be easily undone;</li>
         <li>the control is positioned where it will be difficult to reach, or is near the edge of the screen;</li>
         <li>the control is part of a sequential task.</li>
   	  </ul>
-      <p>The targets on a screen can have different purposes and uses, and this success criterion specifies how each is to be handled.</p>
+      <p>The targets on a screen can have different purposes and uses, and this Success Criterion specifies how each is to be handled.</p>
```

Comment:
we do not put this in upper case when used generically
```suggestion
      <p>The targets on a screen can have different purposes and uses, and this success criterion specifies how each is to be handled.</p>
```